addons/mail/static/src/core/web/activity_service.js

```diff
@@ -52,7 +52,12 @@ export class ActivityService {
         return action;
     }
 
-    async edit(activityId) {
+    async edit(activityId, resModel, resIds) {
+        const context = {
+            default_res_model: resModel,
+            default_res_id: resIds[0],
+        };
```

Comment:
`res_model` and `res_id` can be deduced from the fields `activity.res_id` and `activity.res_model`.

So the fix could simply be:
```js
const activity = this.store.Activity.get(activityId);
const context = { default_res_model: activity.res_model, default_res_id: activity.res_id };
```

Reply:
Thanks for the review @alexkuhn 🙂 you're right this way It's simpler, I thought about it but tried to do something similar to schedule instead 😅